### PR TITLE
Rewrite database seeder

### LINKED ISSUE #54: Backward Resolvers for RequestType and Request
### Tasks
+ Modify Database Schema
  + Add `RequestGroup` id to `RequestType`
  + Add `RequestType` id to `Request`
+ Write backward GraphQL resolvers
  + `RequestType` resolver should be able to return `RequestType`'s parent `RequestGroup`
  + `Request` resolver should be able to return `Request`'s parent `RequestType`

### LINKED ISSUE #55: Rewrite Database Seeder
### Context
Database schema will be modified (coordinate with @guy-ro )
  + Add `RequestGroup` id to `RequestType`  (`RequestGroup` is the parent of `RequestType`)
  + Add `RequestType` id to `Request` (`RequestType` is the parent of `Request`)

### Task
+ Rewrite seeder to include new fields

### PR DESCRIPTION
**Pull request by jlight99**
This PR addresses #55 
To test: run `npm run seed` from the server/ subdirectory, then go to MongoDB Atlas to see the data with the new fields (add RequestGroup id to RequestType, RequestType id to Request)

### CODE REVIEW

File: server/graphql/resolvers.ts [hunk 1]
```diff
@@ -35,14 +35,16 @@ const resolvers = {
     requestTypes: (parent, __, { dataSources }): Array<RequestTypeInterface> => parent.requestTypes.map(id => dataSources.requestTypes.getById(Types.ObjectId(id)))
   },
   RequestType: {
-    numOpen: (parent, __, { dataSources }): Number => parent.requests.open.length,
+    requestGroup: (parent, __, { dataSources }): RequestGroupInterface => dataSources.requestGroups.getById(Types.ObjectId(parent.requestGroup)),
```
Comment (meganniu) [inline server/graphql/resolvers.ts:38]: @guy-ro is this PR also for the backward resolvers?

File: server/models/requestModel.ts [hunk 2]
```diff
@@ -14,6 +14,9 @@ interface RequestInterface {
 type RequestDocument = RequestInterface & Document;
 
 const requestSchema = new Schema({
+  requestType: {
```
Comment (meganniu) [inline server/models/requestModel.ts:18]: Could we get `requestType` added to `RequestInterface`?

File: server/models/requestTypeModel.ts [hunk 3]
```diff
@@ -16,6 +16,9 @@ interface RequestTypeInterface {
 type RequestTypeDocument = RequestTypeInterface & Document
 
 const RequestTypeSchema = new Schema({
+  requestGroup: {
```
Comment (meganniu) [inline server/models/requestTypeModel.ts:20]: Could we get `requestGroup` added to `RequestTypeInterface`?

File: server/scripts/seed.ts [hunk 4]
```diff
@@ -16,6 +16,69 @@ import { RequestType } from '../models/requestTypeModel'
...
+      open: requestIDsForType
+    },
+  })
+  return createSavePromise(type, errMsg)
+}
+
+const createRequestGroup = (groupID, typeIDs, errMsg) => {
+  const group = new RequestGroup({
+    _id: groupID,
+    name: faker.commerce.department(),
```
Comment (meganniu) [inline server/scripts/seed.ts]: Additional request: (this can be in another PR if u want)

Could we choose the RequestGroup name from [this list](https://docs.google.com/document/d/1sC4j4bU41gpVYu9Q_AOO_0EvV4_pVlAbMfWXJshOEMQ/edit)?

Also could we choose the RequestGroup image from this set?
- https://source.unsplash.com/RcgiSN482VI
- https://source.unsplash.com/7ydep8OEvbc
- https://source.unsplash.com/0hiUWSi7jvs

**Review by meganniu — APPROVED** [review]
LGTM! amazing work 🎉🎉just a few comments

File: server/graphql/resolvers.ts (hunk 1 above)
Comment (guy-ro) [reply server/graphql/resolvers.ts:38]: They are in here (they had to be for testing). So I guess yes

File: server/models/requestModel.ts (hunk 2 above)
Comment (guy-ro) [reply server/models/requestModel.ts:18]: Added

File: server/models/requestTypeModel.ts (hunk 3 above)
Comment (guy-ro) [reply server/models/requestTypeModel.ts:20]: Added

File: server/scripts/seed.ts (hunk 4 above)
Comment (guy-ro) [reply server/scripts/seed.ts]: Made these changes

**Review by guy-ro — APPROVED** [review]
lgtm! I'll merge it in now
